#--- a/components/omnibox/browser/autocomplete_result.cc	2019-08-09 16:47:46.000000000 +0200
#+++ b/components/omnibox/browser/autocomplete_result.cc	2019-08-29 10:29:46.066398612 +0200
#@@ -45,12 +45,13 @@ struct MatchGURLHash {
# 
# // static
# size_t AutocompleteResult::GetMaxMatches() {
#-  constexpr size_t kDefaultMaxAutocompleteMatches = 6;
#+  constexpr size_t kDefaultMaxAutocompleteMatches = 30; //was 6, this needs to be double the value of default_max_matches_per_provider from components/omnibox/browser/omnibox_field_trial.cc
# 
#-  return base::GetFieldTrialParamByFeatureAsInt(
#-      omnibox::kUIExperimentMaxAutocompleteMatches,
#-      OmniboxFieldTrial::kUIMaxAutocompleteMatchesParam,
#-      kDefaultMaxAutocompleteMatches);
#+  return //base::GetFieldTrialParamByFeatureAsInt(
#+//      omnibox::kUIExperimentMaxAutocompleteMatches,
#+//      OmniboxFieldTrial::kUIMaxAutocompleteMatchesParam,
#+      kDefaultMaxAutocompleteMatches//);
#+  ;
# }
# 
# AutocompleteResult::AutocompleteResult() {
diff --git a/components/omnibox/browser/autocomplete_result.cc b/components/omnibox/browser/autocomplete_result.cc
index 9fafcd3863ce..8cf090e85ffb 100644
--- a/components/omnibox/browser/autocomplete_result.cc
+++ b/components/omnibox/browser/autocomplete_result.cc
@@ -50,7 +50,8 @@ size_t AutocompleteResult::GetMaxMatches(bool is_zero_suggest) {
   if (is_zero_suggest)
     return kDefaultMaxAutocompleteMatches;
 #else
-  constexpr size_t kDefaultMaxAutocompleteMatches = 6;
+  constexpr size_t kDefaultMaxAutocompleteMatches = 26; //was 6, this needs to be double the value of default_max_matches_per_provider from components/omnibox/browser/omnibox_field_trial.cc
+  return kDefaultMaxAutocompleteMatches;
 #endif  // defined(OS_ANDROID)
 
   return base::GetFieldTrialParamByFeatureAsInt(
--- a/components/omnibox/browser/omnibox_field_trial.cc	2019-08-09 16:47:46.000000000 +0200
+++ b/components/omnibox/browser/omnibox_field_trial.cc	2019-08-29 10:30:37.455752560 +0200
@@ -352,7 +352,8 @@ void OmniboxFieldTrial::GetDemotionsByTy
 
 size_t OmniboxFieldTrial::GetProviderMaxMatches(
     AutocompleteProvider::Type provider) {
-  size_t default_max_matches_per_provider = 3;
+  size_t default_max_matches_per_provider = 13; // was 3; this needs to be half the value of kDefaultMaxAutocompleteMatches from components/omnibox/browser/autocomplete_result.cc
+  return default_max_matches_per_provider;
 
   std::string param_value = base::GetFieldTrialParamValueByFeature(
       omnibox::kUIExperimentMaxAutocompleteMatches,
